doc/api/thaliemitter.md

```diff
@@ -0,0 +1,189 @@
+# The `ThaliEmitter` class
+
+The `ThaliEmitter` class is a bridge between the native Cordova plugin and JavaScript to give you an easy to use interface for broadcasting availability and to connect and disconnect from peer clients.
+
+## Usage
+
+This is the basic usage to listen for the `peerAvailabilityChanged` event as well as start broadcasting the availability of the current device.
```

Comment:
You are assuming that the reader has actually read replication.md. Which is fine but seems like the kind of thing that should be explicitly pointed out in the intro.